Jenkinsfile

```diff
@@ -91,7 +91,7 @@ pipeline {
 		}
 	}
 	environment {
-		run_ci = sh(script: "git fetch origin ${env.CHANGE_TARGET} && git diff --name-only origin/${env.CHANGE_TARGET} | egrep ${tested_files}", returnStatus: true)
+		run_ci = sh(script: "git fetch origin ${env.CHANGE_TARGET} && git diff --name-only origin/${env.CHANGE_TARGET} | egrep \"${tested_files}\"", returnStatus: true)
```

Comment:
:+1: I was wondering what was going on with L4, but when I scrolled down this made more sense. :)

Reply:
Oh this looks way easier to maintain now. Thanks!